Source/Reporting/Web/src/app/Reporting/CaseReportsForListing/case-report-list.component.scss

```diff
@@ -52,6 +52,19 @@ th {
     }
 }
 
+.table>tbody>tr>td,
+.table>tbody>tr>th {
+  border-top: none;
+}
+tr:hover {
```

Comment:
This will hit all tr. Should maybe together with the selector above become:
.className {
   tr { ... }  
   td, th { ... }
}